### PR TITLE
Update docs about `zapier convert`

### PR DESCRIPTION
**Pull request by eliangcs**

### CODE REVIEW

File: README-source.md [hunk 1]
```diff
@@ -269,18 +269,19 @@ zapier deprecate 1.0.0 2017-01-01
 
 ## Converting an Existing App
 
-> Warning! This is in a **very** alpha state - the immediate goal is to provide some basic structure to match an existing application. It will not even get close to a working copy of your existing app (yet).
-
-If you have an existing application (nominally named "V2") on https://zapier.com/developer/builder/ you can use it as a template to kickstart your local application.
+If you have an existing Web Build app on https://zapier.com/developer/builder/ you can use it as a template to kickstart your local application.
```
Comment (xavdid) [inline README-source.md]: `Web Build` -> `Web Builder`. 

Also I would probably hide the developer/builder link in markdown. Maybe something like `If you have an existing web builder app on the [Zapier site](https://zapier.com/developer/builder/), you can ...`

File: README-source.md [hunk 2]
```diff
@@ -269,18 +269,19 @@ zapier deprecate 1.0.0 2017-01-01
...
 ```bash
-# the 1234 is from the URL in https://zapier.com/developer/builder/
-zapier convert 1234 .
+# Convert an existing Web Builder app to CLI in my-app directory
+# App ID 1234 is from URL https://zapier.com/developer/builder/
+zapier convert 1234 my-app
 ```
 
-You app will be created and you can continue working on it.
+You CLI app will be created and you can continue working on it.
```
Comment (xavdid) [inline README-source.md]: `you` -> `your`

File: README-source.md [hunk 3]
```diff
@@ -269,18 +269,19 @@ zapier deprecate 1.0.0 2017-01-01
...
-> Warning! This is in a **very** alpha state - the immediate goal is to provide some basic structure to match an existing application. It will not even get close to a working copy of your existing app (yet).
-
-If you have an existing application (nominally named "V2") on https://zapier.com/developer/builder/ you can use it as a template to kickstart your local application.
+If you have an existing Web Build app on https://zapier.com/developer/builder/ you can use it as a template to kickstart your local application.
 
 ```bash
-# the 1234 is from the URL in https://zapier.com/developer/builder/
-zapier convert 1234 .
+# Convert an existing Web Builder app to CLI in my-app directory
+# App ID 1234 is from URL https://zapier.com/developer/builder/
```
Comment (xavdid) [inline README-source.md]: Give a full url instead. 

`The App ID is a series of numbers near the end of your web builder link: https://zapier.com/developer/builder/app/1234/development`

File: README-source.md [hunk 4]
```diff
@@ -269,18 +269,19 @@ zapier deprecate 1.0.0 2017-01-01
...
 
-> Warning! This is in a **very** alpha state - the immediate goal is to provide some basic structure to match an existing application. It will not even get close to a working copy of your existing app (yet).
-
-If you have an existing application (nominally named "V2") on https://zapier.com/developer/builder/ you can use it as a template to kickstart your local application.
+If you have an existing Web Build app on https://zapier.com/developer/builder/ you can use it as a template to kickstart your local application.
 
 ```bash
-# the 1234 is from the URL in https://zapier.com/developer/builder/
-zapier convert 1234 .
+# Convert an existing Web Builder app to CLI in my-app directory
```
Comment (xavdid) [inline README-source.md]: * `to CLI` -> `into a CLI app`
* `in my-app directory` -> `in the my-app directory`

**Comment by eliangcs** [comment]
Can't believe I didn't see those typos. Thanks! Fixed in 1ca65b4.

**Pull request merged**